### ISSUE TITLE
Show Blocked Categories on Thumbnails as an icon in Categories page.

### ISSUE THREAD

**Issue by Fujosej** [FrankerFaceZ/FrankerFaceZ#1088]
**Is your feature request related to a problem? Please describe.**
Would like to see on this page , what categories are blocked currently.
![image](https://user-images.githubusercontent.com/37980944/127472599-4810df3b-5364-4628-baf2-3418c0c1fbf9.png)
I have lots of categories blocked. So it would be nice to have a visual guide to quickly determine which of the categories are blocked.
![image](https://user-images.githubusercontent.com/37980944/127472375-0fed4ded-32b2-4b2c-b703-d967a9925e0b.png)

**Describe the solution you'd like**
I think one of the easiest solutions , is to show some kind of a visual icon on a category thumbnail

![image](https://user-images.githubusercontent.com/37980944/127473065-34a0814e-cc54-461c-96ff-03d375aabe9e.png)  

**Additional context**
It's really not that big or important feature , but from time to time when i decide to click on categories , it reminds me that it would be nice to quickly see blocked categories, maybe even unblock them from there?
